bot/seasons/evergreen/trivia_quiz.py

```diff
@@ -0,0 +1,238 @@
+                start_embed.title = "Quiz game Starting!!"
+                start_embed.description = "Each game consists of 5 questions.\n"
+                start_embed.description += "**Rules :**\nNo cheating and have fun!"
+                start_embed.set_footer(
+                    text="Points for that question reduces by 25 after 10s.Total time is 30s per question"
+                )
+                await ctx.send(embed=start_embed)  # send an embed with the rules
+                await asyncio.sleep(1)
+
+        elif option == "stop":
+            if self.game_status[ctx.channel.id] is False:
+                await ctx.send("No game running, nothing to stop here.")
+                return
+            else:
+                if (
+                        ctx.author == self.game_owners[ctx.channel.id]
+                        or Roles.moderator in [role.id for role in ctx.author.roles]
+                ):
+                    await self.declare_winner(ctx.channel, player_data)
+                    self.game_status[ctx.channel.id] = False
+                    del self.game_owners[ctx.channel.id]
+                else:
+                    await ctx.send(f"{ctx.author.mention}, you are not authorised to stop this game :ghost: !")
+
+        if category not in self.categories:
+            embed = self.category_embed
+            await ctx.send(embed=embed)
+            return
+        topic = self.questions[category]
+
+        unanswered = 0
+        done_question = []
+        hint_no = 0
+        answer = None
+        hints = None
+        while self.game_status[ctx.channel.id] is True:
+            if len(done_question) > self.question_limit and hint_no == 0:
+                await ctx.send("The round ends here.")
+                await self.declare_winner(ctx.channel, player_data)
+                break
+            if unanswered > 3:
+                await ctx.send("Game stopped due to inactivity.")
+                await self.declare_winner(ctx.channel, player_data)
+                break
+            if hint_no == 0:
+                while True:
+                    question_dict = random.choice(topic)
+                    if question_dict["id"] not in done_question:
+                        done_question.append(question_dict["id"])
+                        break
+                q = question_dict["question"]
+                answer = question_dict["answer"]
+
+                embed = discord.Embed(colour=discord.Colour.gold())
+                embed.title = f"Question #{len(done_question)}"
+                embed.description = q
+                await ctx.send(embed=embed)
+
+            def check(m: discord.Message) -> bool:
+                ratio = fuzz.ratio(answer.lower(), m.content.lower())
+                return ratio > 80 and m.channel == ctx.channel
+            try:
+                msg = await self.bot.wait_for('message', check=check, timeout=10)
+            except Exception as e:
```

Comment:
This shouldn't be a general exception. Add the exceptions you're expecting to be raised here so you avoid suppressing unexpected errors.